src/gui/qgsorderbydialog.cpp

```diff
@@ -71,6 +72,8 @@ QgsFeatureRequest::OrderBy QgsOrderByDialog::orderBy()
       if ( nullsFirstIndex == 1 )
         nullsFirst = true;
 
+      if ( !isExpression ) expressionText = QgsExpression::quotedColumnRef( expressionText );
```

Comment:
Can you add a line break?

    if ( !isExpression )
      expressionText = QgsExpression::quotedColumnRef( expressionText );